contracts/scripts/deployments/DeploySpace.s.sol

```diff
@@ -39,6 +39,9 @@ import {DeployMembershipReferral} from "contracts/scripts/deployments/facets/Dep
 import {DeployMultiInit} from "contracts/scripts/deployments/DeployMultiInit.s.sol";
 
 contract DeploySpace is DiamondDeployer {
+  address internal constant GOVERNANCE_ADDRESS =
+    0x63217D4c321CC02Ed306cB3843309184D347667B;
```

Comment:
Is this the same contract address on base sepolia as on base mainnet ?

Reply:
That's correct, both are the same